src/frontend/tests/core/features/folders.spec.ts

```diff
@@ -109,7 +112,7 @@ test("add a flow into a folder by drag and drop", async ({ page }) => {
     dataTransfer,
   });
 
-  await page.waitForTimeout(3000);
+  await page.waitForTimeout(1000);
 
   const genericNode = page.getByTestId("div-generic-node");
```

Comment:
```suggestion
const genericNode = await page.waitForSelector('[data-testid="div-generic-node]', { timeout: 3000 });
```

Reply:
i've tried this one, but for some reason this does not work, that's why i left this one there